docs/development/classlib/classlib-manual.rst

```diff
@@ -0,0 +1,72 @@
+.NET Core Class Library Manual
+==============================
+
+.NET Core is a modern implementation of .NET. It includes many familiar .NET APIs, which are supported on Linux, OS X, Windows and other operating systems. It is both an implementation but also a standard API definition that can (and is) supported by other versions of .NET. 
```

Comment:
What does '.NET Core library' even mean? From my point of view PCL and .NE Core class libraries are the same.